src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -41,10 +47,28 @@ object AutoExcuse : Module() {
         listener<PacketEvent.Receive> {
             if (mc.player == null || it.packet !is SPacketUpdateHealth) return@listener
             if (it.packet.health <= 0f && timer.tick(3L)) {
-                MessageSendHelper.sendServerMessage(getExcuse())
+                    MessageSendHelper.sendServerMessage(getExcuse())
             }
         }
     }
 
+    override fun onToggle() {
+        if (mode.value == Mode.READ_FROM_FILE) {
+            if (file.exists() && file.length() != 0L) {
```

Comment:
please filter spaces and make sure messages contain more then a space, and the line isn't empty (*just copy spammer. if you want, I'll turn this into a proper util*)

This is because you are kicked for sending empty messages and not everyone is using AntiPacketKick